app/Models/Context.php

```diff
@@ -177,61 +229,59 @@ public static function updateUsingRequest() {
 		self::$order = Minz_Request::param(
 			'order', self::$user_conf->sort_order
 		);
-		self::$number = intval(Minz_Request::param('nb', self::$user_conf->posts_per_page));
+		self::$number = (int)Minz_Request::param('nb', self::$user_conf->posts_per_page);
```

Comment:
In PHPStan level 9, such changes introduce a new error: `Cannot cast mixed to int.`

Reply:
Although `intval()` and `(int)` might be doing exactly the same thing (I have not checked), there are semantically not exactly equivalent